docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -487,6 +487,141 @@ LARGEINT if the precision of the argument is greater than or equal to
 ABS (-20 + 12)
 ```
 
+<<<
+[[aes_encrypt_function]]
+== AES_ENCRYPT Function
+ 
+The AES_ENCRYPT function encrypts a `_string_` using a specified encryption `_key_` with the AES (Advanced Encryption Standard) algorithm and returns a binary string. 
+
+This function prevents sensitive data from being visible to the public. For example, use the AES_ENCRYPT function to encrypt person identity number to preserve user confidentiality.
+
+`AES_ENCRYPT (_string_, _key_)`
+
+[[syntax_description_of_aes_encrypt]]
+=== Syntax Description of AES_ENCRYPT
+
+* `_string_` 
++
+is the data to be encrypted. 
+
+* `_key_` 
++
+is the encryption key to encrypt the `_string_`.
```

Comment:
Later you give an example on using a cryptographic hash function to encrypt the key.  Perhaps you can mention it here ->   "key" can be hashed using a cryptographic hash functions such as SHA1 and MD5.  
Do you know what cryptographic functions can be specified?

Reply:
1. Thanks Roberta, I've incorporated your comments.

2. SHA (SHA1), SHA2 and MD5 can be specified, I've added the following contents:

> This _key_ can be hashed using a cryptographic hash functions, such as SHA, SHA2 and MD5.
>>Example
>>>* This example demonstrates how the AES_ENCRYPT function works with the SHA function.
>>>>
>>>>```
>>>>SELECT AES_ENCRYPT ('hello',SHA ('world')) FROM DUAL;
>>>>(EXPR)
>>>>----------------
>>>>[}âXéò0Êw«æÝAÆc
>>>>--- 1 row(s) selected.
>>>>```

>>>* This example demonstrates how the AES_ENCRYPT function works with the SHA2 function.
>>>>```
>>>>SELECT AES_ENCRYPT ('hello',SHA2 ('world',512)) FROM DUAL;
>>>>(EXPR)
>>>>----------------
>>>>¤÷èx¼ó=L*Ì-µO*A
>>>>--- 1 row(s) selected.
>>>>```

>>>* This example demonstrates how the AES_ENCRYPT function works with the MD5 function.
>>>>```
>>>>SELECT AES_ENCRYPT ('hello',MD5 ('world')) FROM DUAL;
>>>>(EXPR)
>>>>----------------
>>>>C¡ïcåsí[1Ø
>>>>--- 1 row(s) selected.
>>>>```

Reply:
Looks good